src/bidiMapper/domains/context/browsingContextImpl.ts

```diff
@@ -190,6 +190,17 @@ export class BrowsingContextImpl {
     return this.#parentId === null;
   }
 
+  get top(): BrowsingContextImpl {
```

Comment:
The previous method could return `null` or `undefined`.

What happens here if you call `<some context>.top().top()`? The parent of a top-level context should be `null` (c.f. `isTopLevelContext()`).